### PR TITLE
Sarah's awesome new guides!

### PR DESCRIPTION
**Pull request by FredKSchott**
@sarah11918 as discussed I took a pass on the docs and made some edits. I tried to take notes as I went on the larger concepts that my edits were based on, which I'll attach below. But overall these were already in a really great place for a first draft!

As I edited I realized that the docs are all referencing each other in interesting ways, so I ended up working on them all in one branch. I'm happy to break this up and merge these back into your individual PRs if you'd like, or we can just do our final reviews in this one PR, all together. I'll leave the choice up to you!

## Key takeaways

1. **The more examples, the better!** I tried to add a simple example to every section where it made sense, and removed as much as possible from existing examples that could be a distraction. I'm a big believer in most readers only skim examples, so I definitely recommend adding examples to illustrate concepts whenever it makes sense.
1. **Small, focused examples are the best.** For example (ha!) there was one example in the project layout page that used Sass in the style without a clear reason. I converted this to CSS to simplify and focus the example on project layout stuff.
1. **Assume the worst (in walkthroughs)** Its good to add some language around "if it worked, you will see" or "if it didn't work, do X" to walkthroughs like the installation guide, so that people can check in their progress before continuing. 
1. **Every page needs 1+ sentence at the top, introducing the page.** Who is it for, what is the overall page purpose, etc. The reason I say "need" is that our table of contents assumes that one always exists and shows an "Overview" header that points to it.
1. **Odd HTML elements?** Just a note to watch out for `<br>` and `<code>` elements sneaking in from whatever editor you're using :) one of the pages still had a bunch of these. Not a big deal at all, but good to avoid.

### CODE REVIEW

**Comment by sarah11918** [comment]
This is now back in your hands @FredKSchott !

As a reminder, if wanting to put this content live into current docs (not just merge these additional -new files into main): 

- most of the files here are currently NEW files (named -new) and do not replace any existing content in main right now. The existing files will need to be deleted and these files will need renaming to replace the old files.

- manual-setup should be added to the sidebar in config.ts

- Quick Start should be removed from the sidebar in config.ts because it is redundant

File: src/pages/en/core-concepts/project-structure-new.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+You can place CSS and JavaScript in your `public/` directory, but be aware that those files will not be bundled or optimized in your final build. 
+
+ 💡 *As a general rule, any CSS or JavaScript that you write yourself should live in your `src/` directory.*
+
+### `package.json`
+
+This is a file used by JavaScript package managers to manage your dependencies. It also defines the scripts that are commonly used to run Astro (ex: `npm start`, `npm run build`).
+
+For help creating a new `package.json` file for your project, check out the [manual setup](/en/guides/manual-setup-new) instructions.
```
Comment (sarah11918) [inline src/pages/en/core-concepts/project-structure-new.md:99]: When these pages go live and replace the current versions, presumably the file name will change to manual-setup and this link should be updated.

File: src/pages/en/installation-new.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+npm init astro
+
+# yarn
+yarn create astro
+
+# pnpm
+pnpm create astro
+```
+
+> ⚔️ Prefer to go it alone? Read our [manual setup](/en/guides/manual-setup-new) instructions instead.
```
Comment (sarah11918) [inline src/pages/en/installation-new.md:30]: When these pages replace the existing versions and lose their "-new", this link should be updated.

File: src/pages/en/core-concepts/astro-pages-new.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+---
+// Example: src/pages/index.astro
+import MySiteLayout from '../layouts/MySiteLayout.astro';
+---
+<MySiteLayout>
+  <p>My page content, wrapped in a layout!</p>
+</MySiteLayout>
+```
+
+📚 Read more about [layout components](/en/core-concepts/layouts-new) in Astro.
```
Comment (sarah11918) [inline src/pages/en/core-concepts/astro-pages-new.md:47]: Another -new link to be updated before going live

File: src/pages/en/guides/manual-setup-new.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+- public/
+  - robots.txt
+- package.json
+- package-lock.json (or: yarn.lock, pnpm-lock.json, etc.)
+- node_modules/
+```
+
+Congratulations, you're now set up to use Astro!
+
+If you followed this guide completely, you can jump directly to [Step 3: Start](/en/installation-new#3-start-) to continue and learn how to run Astro for the first time.
```
Comment (sarah11918) [inline src/pages/en/guides/manual-setup-new.md:102]: Another -new link to be updated before going live

**Comment by FredKSchott** [comment]
@sarah11918 if these look good to you, then they look good to me! Lets go ahead and update the docs in place, I'll make that change to this PR now.

**Comment by sarah11918** [comment]
@FredKSchott I'm happy with them! LGTM. YOLO.

**Review by sarah11918 — APPROVED** [review]

**Pull request merged**